clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -83,9 +83,9 @@ private static String toHtml() {
         b.append("<th>Key</th>\n");
         b.append("</tr>");
         for (ApiKeys key : ApiKeys.values()) {
-            b.append("<tr>\n");
+        	b.append("<tr>\n");
             b.append("<td>");
-            b.append(key.name);
+            b.append("<a href=\"protocol_api_keys.html#"+ key.name +"\">" + key.name + "</a>");
```

Comment:
I don't think a reference to `protocol_api_keys.html` is required here; because that file is loaded as a server side include (SSI) inside `protocol.html`. I would prefix the anchor labels with something like `The_Messages` (which is the referred main section name) instead to make them uniform. The hyperlinks should work fine after fixing this.